isis/src/base/apps/bandnorm/tsts/batchlist/Makefile

```diff
@@ -1,7 +0,0 @@
-APPNAME = bandnorm
-
-include $(ISISROOT)/make/isismake.tsts
-
-commands:
-	$(APPNAME) from=$(INPUT)/\$$\1.cub \
-	  to=$(OUTPUT)/\$$\1.norm.cub average=band -batchlist=$(INPUT)/testInput.lis > /dev/null;
```

Comment:
It'd be nice to know the context of this test. Seems like an odd place for a batchlist test

Reply:
yeah, I just ignored it, I imagine we can have dedicated batchlist stuff elsewhere but I do not know what would be the point of testing that app specifically.

Reply:
So apparently related to this: https://github.com/USGS-Astrogeology/ISIS3/issues/1246 

So my hunch of it being something addressing an old problem that isn't relevant anymore was spot on. I don't beleive it's worth testing and if we do it'll be somewhere else.